modules/drivers/druid-jdbc/deps.edn

```diff
@@ -0,0 +1,5 @@
+{:paths
+ ["src" "resources"]
+
+ :deps
+ {org.apache.calcite.avatica/avatica  {:mvn/version "1.23.0"}}}
```

Comment:
it would be nice to add a little comment next to this dep like we do for the deps in the main project `deps.edn`, I had to google Apache Calcite Avatica to figure out what this is for and why you're including it. Makes sense but it will save people a step in the future

Reply:
I've added link to docs and small description here.